docker/sqs-docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3'
+services:
+  sqs:
+    image: roribio16/alpine-sqs:latest
```

Comment:
is this used for manual local testing? or for application unit tests?

Reply:
yes, we will remove it and use just for local testing.

Reply:
ok thank you